### ISSUE TITLE
Helper command to print contract addresses for a network

### ISSUE THREAD

**Issue by ferrouswheel** [singnet/snet-cli#215]
Instead of having to look up documentation it'd be nice to just make it easy to print that information since the snet-cli has that knowledge already.

**Comment by astroseger** [comment]
In ```snet session```?

**Comment by ferrouswheel** [comment]
yeah, that seems like the best place, probably under the network section as `default_registry_address` etc

```
$ snet session
    identity.joel:
        default_wallet_index: '0'
        identity_type: key
    ipfs:
        default_ipfs_endpoint: http://ipfs.singularitynet.io:80
    network.ropsten:
        default_eth_rpc_endpoint: https://ropsten.infura.io
        default_gas_price: '1000000000'
    session:
        identity: joel
        network: ropsten
```

**Comment by astroseger** [comment]
was fixed by #232
